django/thunderstore/api/cyberstorm/views/community_filters.py

```diff
@@ -0,0 +1,34 @@
+from rest_framework.generics import get_object_or_404
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from thunderstore.api.cyberstorm.serializers import (
+    CyberstormPackageCategorySerializer,
+    CyberstormPackageListingSectionSerializer,
+)
+from thunderstore.api.utils import CyberstormAutoSchemaMixin
+from thunderstore.community.models import Community
+
+
+class CommunityFiltersAPIView(CyberstormAutoSchemaMixin, APIView):
+    """
+    Return info about PackageCategories and PackageListingSections so
+    they can be used as filters.
+    """
+
+    def get(self, request, community_id: str, format=None):
+        queryset = Community.objects.prefetch_related("package_categories")
```

Comment:
Not a blocker by any means, but any reason you didn't make this a class variable like some other views do it? Should work just fine even if we're not extending a base view that utilizes it automatically I think.

Reply:
TBH DRF docs make it hard for me to know what's the preferred way of doing things and what are the consequences of choosing between them. E.g. I didn't want to use `get_queryset` since that would end up being a function call that makes one function call, requiring the reader to jump between function definitions. Didn't consider inheritance at that point.

Reply:
Yeah I much prefer the clarity here over weird inheritance logic too. I just meant that the queryset could be a class variable and referred to with `self.queryset` rather than re-defining it for every request, so not really much to do with DRF (aside for how that could conflict with some of its default behavior).